backend/director/llm/__init__.py

```
@@ -12,12 +13,15 @@ def get_default_llm():
 
     openai = True if os.getenv("OPENAI_API_KEY") else False
     anthropic = True if os.getenv("ANTHROPIC_API_KEY") else False
+    xai = True if os.getenv("XAI_API_KEY") else False
```

Ruff (0.8.0)

Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)